packages/browser/src/core/user/__tests__/index.test.ts

```diff
@@ -865,19 +851,36 @@ describe('group', () => {
 })
 
 describe('store', function () {
-  const store = new LocalStorage()
-  beforeEach(function () {
-    clear()
-  })
-
   describe('#get', function () {
-    it('should not not get an empty record', function () {
-      expect(store.get('abc') === undefined)
+    it('should throw an error', function () {
```

Comment:
"should return null if localStorage throws" would be an improved description

Reply:
Agreed!